api/src/main/resources/liquibase.xml

```diff
@@ -720,4 +720,19 @@
         <comment>Renaming column column_number to bed_column_number in table bed_location_map because column_number is a reserved word in MySQL 8.0.2 and later</comment>
         <renameColumn columnDataType="int" newColumnName="bed_column_number" oldColumnName="column_number" tableName="bed_location_map" />
     </changeSet>
+
+    <changeSet id="BAH-3896_300520241229" author="mohankumar">
```

Comment:
Change Set ID and overall PR should refer to a ticket in the OpenMRS JIRA instance for easier community tracking:  https://openmrs.atlassian.net/browse/BED

Reply:
Have created an OpenMRS issue and updated references to it. https://openmrs.atlassian.net/browse/BED-9